modules/core/02-client/keeper/grpc_query_test.go

```diff
@@ -377,11 +387,11 @@ func (suite *KeeperTestSuite) TestQueryConsensusStates() {
 			if tc.expPass {
 				suite.Require().NoError(err)
 				suite.Require().NotNil(res)
+				suite.Require().Equal(tc.expectedConsensusStates, res.Pagination.Total)
 				suite.Require().Equal(len(expConsensusStates), len(res.ConsensusStates))
```

Comment:
Aren't these checks essentially the same?

Reply:
true actually, i guess this test was passing bc we were reconstructing the situation and did not have other items in the query resp, whereas in real life without `FilterPaginate` other items were being returned in addition.

Reply:
i can add some other items maybe to the update